compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -549,6 +549,106 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager) {
   passManager.addPass(createCanonicalizerPass());
 }
 
+void addMLIRAIRAIELegacyLoweringPasses(OpPassManager &passManager) {
```

Comment:
What is changed in `addMLIRAIRAIELegacyLoweringPasses` compared to `addMLIRAIRAIELoweringPasses`?

Reply:
No changes yet, this just de-couples the default pipeline (which will be pad-pack after this commit) from the rest of them, so that we can start changing the lowering in `addMLIRAIRAIELoweringPasses` without worrying about if it will break various pipelines.